src/main/kotlin/com/util/ReservationCalculator.kt

```diff
@@ -0,0 +1,37 @@
+package com.util
+
+import com.model.Reservation
+import com.parkingrate.ParkingRate
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.time.Duration
+
+@Service
+class ReservationCalculator {
+    fun calculateReservationCost(reservation: Reservation,
+                                 parkingRate: ParkingRate): BigDecimal {
+        val duration = Duration.between(reservation.startTime, reservation.endTime)
+        val durationInHoursWithStartedHour = duration.toHours().plus(1)
```

Comment:
jeśli duration wyjdzie jako równa godzina to i tak dodamy godzinę?

Reply:
Racja, mój idiotyczny błąd.